views/recipe-edit.handlebars

```diff
@@ -1,75 +1,102 @@
-                        <select
-                            class="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                            id="recipeTotalTimeUOM" name="recipeTotalTimeUOM" data-value="{{totalTimeUOM}}">
-                            {{#each timeUOMs as |option|}}
-                            <option value="{{option.value}}">{{option.display}}</option>
-                            {{/each}}
-                        </select>
-                    </div>
-                    <div>
-                        <label for="recipeComplexity"
-                            class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Complexity</label>
-                        <input type="text" name="recipeComplexity" id="recipeComplexity"
-                            class="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                            placeholder="Simple" required="" value="{{complexity}}">
+                <form id="recipeform" class="space-y-4 md:space-y-6" data-recipeid="{{id}}" target="_self" method="POST">
+                    <div class="flex-row md:columns-2 justfiy-between">
+                        <div>
+                            <label for="recipeName"
+                                class="block mb-1 text-sm font-medium text-neutral-900 dark:text-white">Recipe Name</label>
+                            <input type="text" name="recipeName" id="recipeName"
+                                class="bg-neutral-50 border border-neutral-300 text-neutral-900 sm:text-sm rounded-lg focus:ring-green-500 focus:border-neutral-300 block w-full p-2.5 dark:bg-neutral-700 dark:border-neutral-600 dark:placeholder-neutral-400 dark:text-white caret-green-500 mb-2"
+                                placeholder="Recipe Name" required="" value="{{name}}">
+                        </div>
+                        <div>
+                            <label for="recipeThumbnail"
+                                class="block mb-1 text-sm font-medium text-neutral-900 dark:text-white">Thumbnail URL</label>
+                            <input type="text" name="recipeThumbnail" id="recipeThumbnail"
+                                class="bg-neutral-50 border border-neutral-300 text-neutral-900 sm:text-sm rounded-lg focus:ring-green-500 focus:border-neutral-300 block w-full p-2.5 dark:bg-neutral-700 dark:border-neutral-600 dark:placeholder-neutral-400 dark:text-white caret-green-500 mb-2"
+                                placeholder="https://" value="{{thumbnail}}">
```

Comment:
thinking this needs to be thumbnailURL, doesn't appear to be rendering correctly on the screen whatever the cause is. 
This recipe has a thumbnail but it didn't load on the form
![image](https://user-images.githubusercontent.com/119005046/236703924-f92b136c-3757-408c-ac49-b0b13c94df7e.png)